cloud/gce/launch.sh

```diff
@@ -2,38 +2,27 @@
 # This is a simple wrapper script around the cockroach binary.
 # It must be in the same directory as the cockroach binary, and invoked in one of two ways:
 #
-# To start the first node and initialize the cluster:
-# ./launch init [load balancer address:port]
 # To start a node:
 # ./launch start [load balancer address:port]
 set -ex
 
 source config.sh
 
-DATA_DIR="data"
-LOG_DIR="logs"
-STORES="ssd=${DATA_DIR}"
-COMMON_FLAGS="--log-dir=${LOG_DIR} --logtostderr=false ${STORES}"
-START_FLAGS="--insecure --addr=${LOCAL_ADDRESS}:${PORT}"
+DATA_DIR="cockroach-data"
+LOG_DIR="cockroach-data/logs"
+COMMON_FLAGS="--log-dir=${LOG_DIR} --logtostderr=false"
+START_FLAGS="--insecure --host=${LOCAL_ADDRESS} --port=${SQL_PORT} --http-port=${HTTP_PORT} --join=${JOIN_ADDRESS}"
 
 action=$1
-if [ "${action}" != "init" -a "${action}" != "start" ]; then
-  echo "Usage: ${0} [init|start] [load balancer address:port]"
+if [ "${action}" != "start" ]; then
```

Comment:
since we only have one possible action now, this can be removed, along with the `action` variable, the call in `main.tf` and the readme.